CHANGELOG.md

```diff
@@ -17,6 +18,39 @@ The below sections record the changes for each API version in each release as fo
 * for subsequent release-candidate(s), only the delta to the previous release-candidate
 * for a public release, the consolidated changes since the previous public release

```

Comment:
One initial remark: Please consider this for the r2.2 release notes: list all changes since the previous public release (yes, that is repeating r2.1 changes)

Reply:
right, I forgot, we did the same in Fall24.